pkg/ovs/openflow/ofctrl_priority.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+type Priority uint16
+
+func NewPriority(val uint16) Priority {
+	return Priority(val)
+}
+
+func (p Priority) Add(offset uint16) Priority {
```

Comment:
btw, do we allow to have negative number of priority? if it has no check here, I guess it might be negative when user pass the offset value as negative number.

Reply:
OK, I thought of reviewers can notice this. I think I need add a check here.